openpype/hosts/max/plugins/publish/collect_render.py

```diff
@@ -42,6 +42,28 @@ def process(self, instance):
 
         instance.data["cameras"] = [camera.name] if camera else None        # noqa
 
+        if instance.data.get("multiCamera"):
+            cameras = instance.data.get("members")
+            if not cameras:
+                raise RuntimeError("There should be at least"
+                                   " one renderable camera in container")
```

Comment:
These types of errors should occur in validation preferably so the user can get presented a clear error instead of "This is not your fault"

Reply:
Also, if you'd still want to error here - this seems like a `KnownPublishError` instead.